src/components/ButtonPanel.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Button from './Button';
+
+const genKey = () => Math.floor(Date.now() * Math.random());
```

Comment:
* [ ] There are packages you could use for efficient id generation, you can try `short-id` or `UUID` packages as they guarantee unique id generation.